### PR TITLE
Adding new version of US daily email, with small images as first image.

### PR DESCRIPTION
**Pull request by crifmulholland**
image

### CODE REVIEW

File: template/macro/2015/layout.html [hunk 1]
```diff
@@ -147,7 +147,7 @@
                         {% set use_thumbnail = True %}
                         {% set show_text = standfirst %}
                     {% else %}
-                        {% if loop.index == 1 and top_thumb %}
+                        {% if loop.index == 1 and top_thumb and not show_small_img %}
```
Comment (desbo) [inline template/macro/2015/layout.html]: should this be `small_first_img`?

Comment (crifmulholland) [inline template/macro/2015/layout.html]: yeah, it should- but it seems I just don't need that. I'll remove it.

**Comment by desbo** [comment]
:+1:

File: template/macro/2015/stories.html [hunk 2]
```diff
@@ -71,7 +71,7 @@
               <td height="5"></td>
             </tr>
             <tr valign="top">
-              {% if more_thumbs and (index == 2 or index == 3) %}
+              {% if (small_first_img and index == 1) or (more_thumbs and (index == 2 or index == 3)) %}
```
Comment (rrees) [inline template/macro/2015/stories.html]: This is going to have to be made sane at some point. Presumably the trail logic should be to declare the number of trails that have thumbnails.

**Pull request merged**